eel/eel-background.c

```diff
@@ -433,8 +433,8 @@ set_root_surface (EelBackground *self,
         self->details->bg_surface = mate_bg_create_surface (self->details->bg, window,
         						    width, height, TRUE);
     }
-
-    if (self->details->bg_surface != NULL)
+    GdkDisplay *display = gdk_screen_get_display (screen);
```

Comment:
indentation for the if clause is misleading, GdkDisplay should start 4 space to the right, exactly under the `s` from line above.

Reply:
look more closely:
`    if (self->details->bg_surface != NULL)`
was removed and replaced by
`    if ((GDK_IS_X11_DISPLAY (display)) && (self->details->bg_surface != NULL))`
in the same position.
It is followed by
`            mate_bg_set_surface_as_root (screen, self->details->bg_surface)`
with the correct indent. A misleading indent btw brings a build warning, though that can easily get buried under all the old warnings in a full build of caja

Reply:
opps, my fault. The line starting with `if` was removed. sorry for that noise.